imports/plugins/core/collections/lib/registerSchema.js

```diff
@@ -1,11 +1,19 @@
 export const Schemas = {}; // populated with all Schemas
 
 /**
- * Register a schema
+ * @file **Reaction Schemas** - Use these methods to register and fetch Reaction Core schemas.
+ * See {@link https://docs.reactioncommerce.com/reaction-docs/master/simple-schema full documentation}.
+ *
+ * @module collections
```

Comment:
I wonder if we should group `schemas` and `collections` independently at the same prominence?